swagger/swagger-generator/generator-core/src/main/java/io/servicecomb/swagger/extend/ModelResolverExt.java

```diff
@@ -29,60 +29,56 @@
 import com.fasterxml.jackson.databind.JavaType;
 
 import io.servicecomb.foundation.common.exceptions.ServiceCombException;
+import io.servicecomb.foundation.common.utils.SPIServiceUtils;
 import io.servicecomb.swagger.converter.property.StringPropertyConverter;
-import io.servicecomb.swagger.extend.property.ByteProperty;
-import io.servicecomb.swagger.extend.property.ShortProperty;
+import io.servicecomb.swagger.extend.property.creator.ByteArrayPropertyCreator;
+import io.servicecomb.swagger.extend.property.creator.BytePropertyCreator;
+import io.servicecomb.swagger.extend.property.creator.InputStreamPropertyCreator;
+import io.servicecomb.swagger.extend.property.creator.PropertyCreator;
+import io.servicecomb.swagger.extend.property.creator.ShortPropertyCreator;
 import io.swagger.converter.ModelConverter;
 import io.swagger.converter.ModelConverterContext;
 import io.swagger.jackson.ModelResolver;
 import io.swagger.models.Model;
 import io.swagger.models.ModelImpl;
-import io.swagger.models.properties.ByteArrayProperty;
 import io.swagger.models.properties.Property;
 import io.swagger.models.properties.StringProperty;
 import io.swagger.util.Json;
 
 public class ModelResolverExt extends ModelResolver {
```

Comment:
in current code MultiPartFile be a model in swagger, but it should not be a model.

Reply:
ok... what are you proposing?